apps/react/docs/docs/02-Installation/02-Option 2 Docker Compose Setup/index.mdx

```diff
@@ -0,0 +1,38 @@
+---
+id: installation-option-2
+title: Option 2 - Docker Compose Setup
+---
+
+# WKS Platform Installation Guide
+
+## Option 2: Running with Docker Compose Images
```

Comment:
Should we rename to: Option2: Local Development Environment
ps: docker compose can be utilized to facilitate the local environment setup. But the local environment setup is the main goal here